.travis.yml

```diff
@@ -1,5 +1,3 @@
-# sudo is enabled because it provides more memory which was needed to run go test -race
-sudo: required
```

Comment:
The `sudo:` key has been deprecated, see https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure